### ISSUE TITLE
[Suggestion] Config to Disable Music System

### ISSUE THREAD

**Issue by ghost** [quiqueck/BetterEnd#350]
### Description

Hi! I'm a big fan of BetterEnd, and I really enjoy most of the mod. That being said, would it be possible to have a config option to disable the music system entirely? The tracks are nice and all, but I'm personally not a big fan of how the music fades out so frequently. The default biome sizes can be quite small at times, meaning each song gets constantly cut-off when exploring.

Even just moving through biomes at a sprinting pace, I'm only getting to hear an average of 20-30 seconds of a song before it fades into another. Flying with an elytra is even worse, and results in the game being completely silent most of the time, since songs are just constantly fading and cutting out. It can be pretty jarring - even immersion-breaking at times.

Not to mention, this feature breaks _any_ custom music added by other mods or resource packs. If there could be a way to turn it off via a client-side config option in the future, that'd be awesome. Plus, it would allow for some extra compatibility as well. Either way, thanks for making the mod. Apart from this gripe I have, it's easily one of my favorites.

**Comment by PolyframeX** [comment]
I completely agree with this! It would be nice if the music system maybe just switches to the vanilla one instead. Because currently the music cuts out randomly, never plays, or breaks audio for other things. It's just a small thing that can really get annoying.

**Comment by gogekka** [comment]
This is an issue I've been having, too. I feel like the music is a neat concept, but it doesn't work well in a procedurally-generated game like Minecraft. It just cuts off so much and totally ruins the vibe of exploring the mod's gorgeous biomes. Probably better to just use the vanilla system.
